frontend/helpers/generateFakeVault.ts

```diff
@@ -96,16 +97,26 @@ export const generateFakeVaultTransaction = function (): VaultTransaction {
     const collateralizationRatio = fakeVault.collateralAmount.multipliedBy(minUnitPrice).toNumber();
     const proximityToLiquidation = liquidationRatio - collateralizationRatio;
 
-    const state = proximityToLiquidation < 0 ? 'liquidatable' : 'not-liquidateable';
+    const state: VaultTransactionState =
+        proximityToLiquidation < 0 ? faker.helpers.randomize(['liquidatable', 'liquidated']) : 'not-liquidatable';
```

Comment:
I am afraid that you will need to have a separate functions for the `*Liquidated` and `*NotLiquidated` states, which will then be randomised inside `generateFakeVaultTransaction`